inbox.js

```diff
@@ -0,0 +1,214 @@
+var cron = require('node-cron'),
+    mongoose = require('mongoose'),
+    url = 'mongodb://localhost/EMAILPANEL',
+    conn = mongoose.connect(url),
+    in_array = require('in_array'),
+    fs = require('fs'),
+    Imap = require('imap'),
+    inspect = require('util').inspect;
+
+var google = require('googleapis'),
+    googleAuth = require('google-auth-library'),
+    OAuth2 = google.auth.OAuth2;
+
+var YOUR_CLIENT_ID = "26456531203-aa7qlroeerjrbaoatql4s5di1pcmir46.apps.googleusercontent.com",
+    YOUR_CLIENT_SECRET = "k0jbwm1g1Ow69WiYE_UKu5q4",
+    YOUR_REDIRECT_URL = "https://developers.google.com/oauthplayground";
+
+var oauth2Client = new OAuth2(YOUR_CLIENT_ID, YOUR_CLIENT_SECRET, YOUR_REDIRECT_URL);
+// Retrieve tokens via token exchange explained above or set them:
+oauth2Client.setCredentials({
+    access_token: "ya29.Gls_BEQkhGIpdPplXqN5rUiACQ4spvr27C02bPtHYp1niYkQPIw-6e3EowzetEr-KWEBEtPkZM5J0KJKcbonkf3fTV_XCjvG5EpYVrl0Ev0I0beLdo0fWoYgKoSq",
+    token_type: "Bearer",
+    expires_in: 3600,
+    refresh_token: "1/vjjHYdLPT7STaQEnv_zXPaWTcpR9dg33oiEbk91m8PE"
+});
+var drive = google.drive({ version: 'v2', auth: oauth2Client });
```

Comment:
create google drive connection in a seperate file maybe 

services/googledrive.js